app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  validates :email, uniqueness: true, presence: true
+  validates_presence_of :password, :role, :active, :address, :city, :state, :zip, require: true
+
```

Comment:
```suggestion
it {should validate_numericality_of(:role).only_integer}
```

As above, we can include tests for the numericality validations in the models. This is for pretty much all numerical inputs but I don't want to put that in every place.

Reply:
Cool, I didn't know this.

Reply:
I moved suggest code to user_spec